content/en/docs/distributions/aws/aws-e2e.md

```diff
@@ -248,16 +248,28 @@ kfctl build -f {{% aws/config-file-aws-cognito %}} -V
 kfctl apply -f {{% aws/config-file-aws-cognito %}} -V
 ```
 
-That shouldn't take a long time. There shouldn't by any errors, and when ready you can validate that you can see the kubeflow namespace.
+That shouldn't take a long time. There shouldn't by any errors, and when ready you can validate that you can see the kubeflow namespace by following command:
```

Comment:
Maybe the last part could be "namespace by using the following command:"

What do you think?